src/V3LinkLevel.cpp

```diff
@@ -257,6 +258,77 @@ void V3LinkLevel::wrapTopCell(AstNetlist* rootp) {
                     pinp->modVarp(oldvarp);
                     cellp->addPinsp(pinp);
                 }
+                else if (instantiateTopIfaces && oldvarp->isIfaceRef()) {
+                    // for each interface port on oldmodp instantiate a corresponding interface cell in $root
+                    if (VN_IS(oldvarp->subDTypep(), IfaceRefDType)) {
+                        AstIfaceRefDType* ifacerefp = VN_AS(oldvarp->subDTypep(), IfaceRefDType);
+                        if (!ifacerefp->cellp()) {
+                            string name = oldvarp->name();
+                            if (dupNames.find(name) != dupNames.end()) {
+                                // __02E=. while __DOT__ looks nicer but will break V3LinkDot
+                                name = oldmodp->name() + "__02E" + name;
+                            }
+
+                            AstCell* ifacecellp = new AstCell(newmodp->fileline(), newmodp->fileline(), name, ifacerefp->ifaceName(), nullptr, nullptr, nullptr);
```

Comment:
```suggestion
                            AstCell* ifacecellp = new AstCell{newmodp->fileline(), newmodp->fileline(), name, ifacerefp->ifaceName(), nullptr, nullptr, nullptr};
```

Use {} for all constructors in new code.

Reply:
Done.